magicctapipe/image/calib.py

```diff
@@ -0,0 +1,102 @@
+
+
+import numpy as np
+
+
+from ctapipe.image import (
+    apply_time_delta_cleaning,   
+    number_of_islands,
+    tailcuts_clean,   
+)
+
+from lstchain.image.cleaning import apply_dynamic_cleaning
+from lstchain.image.modifier import (
+    add_noise_in_pixels,
+    random_psf_smearer,    
+)
+
+__all__ = [
+    "Calibrate_LST", "Calibrate_MAGIC"
+]
+
+def Calibrate_LST(event, tel_id, rng, config_lst, camera_geoms, calibrator_lst, increase_nsb, use_time_delta_cleaning, use_dynamic_cleaning ):
+
+    """
+    This function computes and returns signal_pixels, image, and peak_time for LST
+    """
+    
+    calibrator_lst._calibrate_dl0(event, tel_id)
+    calibrator_lst._calibrate_dl1(event, tel_id)
+
+    image = event.dl1.tel[tel_id].image.astype(np.float64)
+    peak_time = event.dl1.tel[tel_id].peak_time.astype(np.float64)
```

Comment:
Why do you convert to float 64?

Reply:
They are converted to float 64 even in the master. ctapipe can sometimes complain about variables type (in particular float 32 or 64). We can try to remove it but I'm not sure about results and code portability

Reply:
Ok keep it then. I was only curious.